src/test/scala/org/broadinstitute/dsde/agora/server/AgoraTestData.scala

```diff
@@ -639,22 +689,4 @@ object AgoraTestData {
     payload = payloadWithInvalidPersonalDockerTagNameInWdl,
     entityType = Option(AgoraEntityType.Task)
   )
-
-  def populateDatabase(agoraBusiness: AgoraBusiness) = {
```

Comment:
Apparently not. I did a search in IntelliJ and found no references. All the tests pass happily without it.